_health-care/_js/components/Nav.jsx

```diff
@@ -10,85 +11,97 @@ import React from 'react';
 class Nav extends React.Component {
   render() {
     const subnavStyles = 'step one wow fadeIn animated';
+    const completedSections = this.props.data.completedSections;
+    const currentUrl = this.props.currentUrl;
     // TODO(akainic): change this check once the alias for introduction has been changed
     return (
       <ol className="process hca-process">
-        <li className={`one ${subnavStyles} ${this.props.currentUrl.startsWith('/introduction') ? ' section-current' : ''}`}>
+        <li className={`one ${subnavStyles} ${currentUrl.startsWith('/introduction') ? ' section-current' : ''}
+          ${(completedSections['/introduction'] === true) ? ' section-complete' : ''}`}>
```

Comment:
Yeah that sounds good. I am hoping that as we evolve the application that we will have `uiState` structured differently and not have to pass `path`, but for now that looks good.